service/assessment/assessment.go

```diff
@@ -202,3 +368,19 @@ func (s Service) ListAssessmentResults(_ context.Context, _ *assessment.ListAsse
 func (s *Service) RegisterAssessmentResultHook(assessmentResultsHook func(result *assessment.AssessmentResult, err error)) {
 	s.resultHooks = append(s.resultHooks, assessmentResultsHook)
 }
+
+func (s *Service) setEvidenceStoreStream() error {
```

Comment:
should not be called `setXXX` but rather something like init.

Reply:
Right, we will do it only once - and not multiple times